### PR TITLE
[dev-v2.5] 1.19.16-rancher1-5 and 1.20.15-rancher1-3 with nginx v1.2.0

### PR DESCRIPTION
**Pull request by kinarashah**
**Changelog**
--------- 
- Using  `nginx-v1.2.0-rancher1` which is upstream v1.2.0 version bump + template from dev-v2.5 PR. This is used by newly added following versions: 
    - `>=1.19.16-rancher1-5 <1.20.0-rancher1-1`
    - `>=1.20.15-rancher1-3 <1.21.0-rancher1-1` 
    - `>=1.21.12-rancher1-1  <1.22.0-rancher1-1` 
    
https://github.com/rancher/rancher/issues/37390

### CODE REVIEW

**Review by paynejacob — DISMISSED** [review]

**Review by thedadams — DISMISSED** [review]

File: rke/k8s_version_info.go [hunk 1]
```diff
@@ -509,6 +509,14 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRancherVersion: "2.5.11-rc0",
 			MinRKEVersion:     "1.2.14-rc0",
 		},
+		"v1.19.16-rancher1-4": {
```
Comment (snasovich) [inline rke/k8s_version_info.go:512]: Nice catch for adding these missing ones!

File: rke/k8s_rke_system_images.go [hunk 2]
```diff
@@ -5651,6 +5651,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled out of band post v2.5.12
```
Comment (snasovich) [inline rke/k8s_rke_system_images.go]: Nit, but v2.5.13 is out so it should be stating that version I think: https://github.com/rancher/rancher/releases/tag/v2.5.13

File: rke/k8s_rke_system_images.go [hunk 3]
```diff
@@ -6262,6 +6306,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled out of band post v2.5.12
```
Comment (snasovich) [inline rke/k8s_rke_system_images.go]: Same, v2.5.13.

**Review by snasovich — COMMENTED** [review]
One nit, otherwise LGTM (did not review new template though - trust it works).

**Comment by kinarashah** [comment]
Updated from `// Enabled out of band post v2.5.12` to `Enabled out of band post v2.5.13`.

**Review by snasovich — APPROVED** [review]
LGTM
Since the only change is an updated comment we can consider Jacob's and Donnie's approvals still in force, so safe to merge.

**Pull request merged**